server/client/stress_test.go

```diff
@@ -177,45 +175,46 @@ func doTest(t *testing.T, flags []string, numEvents, numSpans, numFrames, concur
 			[]string{
 				"branch=master",
 				filter("duration", duration),
-				filter("events", numEvents),
+				filter("errors", numErrors),
+				filter("transactions", numTransactions),
 				filter("spans", numSpans),
 				filter("frames", numFrames),
-				filter("concurrency", concurrency),
+				filter("numAgents", numAgents),
 				fmt.Sprintf("limit=%d", memLimit)},
 			reports)
 		assert.True(t, ok, msg)
 	}
 }
 
 func TestSmallTransactionsSequential(t *testing.T) {
-	doTest(t, noFlags, "10", "10", "10", "1")
+	doTest(t, noFlags, "10", "10", "10", "10", "1")
```

Comment:
we can look at what kind of v2 tests make sense at other time, but for now maybe just honor test names? eg: pass 0 errors to this one. same thing for the ones bellow